.github/workflows/daily.yml

```diff
@@ -16,6 +16,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Log into quay
+        run: docker login -u="${{ QUAY_BOTUSER }}" -p="${{ secrets.QUAY_TOKEN }}" quay.io
+
+      - name: Synchronize container image mirror
+        run: ./sync-quay
```

Comment:
I wouldn't mind if that was inline for simplicity. You can do the normal YAML thing:
```yaml
    run: |
        shell
        .. script
```
```

Reply:
You mean you want to run it locally as well? Then let's keep it as a script for now indeed. Bikeshedding :-)